src/Files.App/Views/Settings/LayoutPage.xaml

```diff
@@ -200,5 +210,13 @@
 			</local:SettingsBlockControl>
 
 		</StackPanel>
+
+		<!--  Adaptive Layout Disabled Teaching Tip  -->
+		<TeachingTip
+			CloseButtonContent="{helpers:ResourceString Name=OK}"
+			IsLightDismissEnabled="True"
+			IsOpen="{x:Bind ViewModel.ShowAdaptiveDisabledTeachingTip, Mode=TwoWay}"
+			Subtitle="{helpers:ResourceString Name=AdaptiveLayoutDisabledNotification}"
+			Target="{x:Bind LayoutTypeComboBox}" />
```

Comment:
Do you think it would be better to target the sync preferences setting?

Reply:
Yes if the teaching tip message is replaced as you suggest, so I have changed the target.